source/administration/changelog.md

```diff
@@ -4,6 +4,143 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
 
 Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.36 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
+
+**Release Day: 2021-06-16**
+
+### Important Upgrade Notes
+ - Gossip clustering mode is now in General Availability and is no longer available as an option. All cluster traffic will always use the gossip protocol. The config setting ``UseExperimentalGossip`` has no effect and has only been kept for compatibility purposes. The setting to use gossip has been removed from the System Console. **Note:** For High Availability upgrades, all nodes in the cluster must use a single protocol. If an existing system is not currently using gossip, one node in a cluster can't be upgraded while other nodes in the cluster use an older version. Customers must either use gossip for their High Availability upgrade, or customers must shut down all nodes, perform the upgrade, and then bring all nodes back up.
+
+**IMPORTANT:** If you upgrade from a release earlier than v5.35, please read the other [Important Upgrade Notes](https://docs.mattermost.com/administration/important-upgrade-notes.html).
+
+### Highlights
+
+#### Focalboard (Beta)
+ - Focalboard is now integrated with Mattermost as a beta feature in v5.36. It brings easy-to-use project and work management to replace tools such as Trello, Notion, and Asana.
+
+#### Incident Collaboration (Enterprise Edition E20)
+ - Retrospective report, automated welcome message, team-wide playbook access, and permission control for playbook creation.
+
+#### Hungarian Language Support (Beta)
+ - Mattermost is now available in Hungarian.
+
+### Improvements
+
+#### User Interface (UI)
+ - If message autoresponder is set, only one message is now sent to a given user irrespective of how many Direct Message messages the user receives.
+ - Added status icons on **Add members** to channel and **Add members** to team lists.
+ - Added a keyboard shortcut to focus on the Search bar and search in the current channel.
+ - A search tip is now shown when scrolling back in a channel.
+ - Improved the error text in the **Edit Channel Header** modal.
+ - Added the ability to clear a custom status when only an emoji and no text is set.
+ - Redesigned message notification emails.
+ - Status icon was removed from the profile image in the center channel and the right-hand side view.
```

Comment:
This is not accurate. There is an account setting to allow users to do this now, it's not a mandatory change @amyblais